src/Areas/AzureBestPractices/Commands/BestPracticesCommand.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using AzureMcp.Areas.AzureBestPractices.Options;
+using AzureMcp.Commands;
+using AzureMcp.Helpers;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.AzureBestPractices.Commands;
+
+public sealed class BestPracticesCommand(ILogger<BestPracticesCommand> logger) : BaseCommand
+{
+    private const string CommandTitle = "Get Azure Best Practices";
+    private readonly ILogger<BestPracticesCommand> _logger = logger;
+    private static readonly Dictionary<string, string> s_bestPracticesCache = new();
+
+    private readonly Option<string> _resourceOption = BestPracticesOptionDefinitions.Resource;
+    private readonly Option<string> _actionOption = BestPracticesOptionDefinitions.Action;
+
+    public override string Name => "get";
+
+    public override string Description =>
+        "Returns best practices for secure, production-grade Azure usage. " +
+        "Call this command and follow its guidance before generating or suggesting any Azure code. " +
+        "This command returns the content of the best practices file as a string array.";
+
+    public override string Title => CommandTitle;
+
+    protected override void RegisterOptions(Command command)
+    {
+        command.AddOption(_resourceOption);
+        command.AddOption(_actionOption);
+    }
+
+    [McpServerTool(Destructive = false, ReadOnly = true, Title = CommandTitle)]
+    public override Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var resource = parseResult.GetValueForOption(_resourceOption);
+        var action = parseResult.GetValueForOption(_actionOption);
+
+        if (string.IsNullOrEmpty(resource) || string.IsNullOrEmpty(action))
```

Comment:
Please add a Validation handler  I think there is example in BaseClusterCommand.cs

Reply:
Added